recipes/pyfolio/meta.yaml

```diff
@@ -0,0 +1,92 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Could you please remove these comments?

Reply:
Hi @CJ-Wright Do you want me to remove all the comments in the file or only specific ones?